src/server/routes/configurePassport.js

```diff
@@ -17,19 +24,22 @@ export default (passport) => {
           githubId: profile.id,
         })
 
-      if (!users[0]) {
-        const user = await User
+      let user = users[0]
+
+      if (!user) {
+        user = await User
           .query()
           .insert({
             githubId: profile.id,
-            name: profile.displayName,
-            email: profile.emails.find(email => email.primary).value,
+            ...getDataFromProfile(profile),
           })
-
-        done(null, user)
       } else {
-        done(null, users[0])
+        user = await User
+          .query()
+          .patchAndFetchById(user.id, getDataFromProfile(profile))
```

Comment:
Update infos at each login. It seems that github is running a cron to update infos periodically.